core/src/validator.rs

```diff
@@ -1483,7 +1479,7 @@ fn load_blockstore(
     leader_schedule_cache.set_fixed_leader_schedule(config.fixed_leader_schedule.clone());
     {
         let mut bank_forks = bank_forks.write().unwrap();
-        bank_forks.set_snapshot_config(config.snapshot_config.clone());
+        bank_forks.set_snapshot_config(Some(config.snapshot_config.clone()));
```

Comment:
Same as other comment - `load_bank_forks()` can succeed without a `SnapshotConfig` so we must set it later and left it out of this PR